File: Baekjoon/Class2/09.1546.py
import sys
input = sys.stdin.readline

n = int(input())
scores = map(int, input().split())
m = 0
total = 0
for score in scores:
    total += score
    if score > m:
        m = score
print((total / m * 100) / n)

'''
n = int(input())
scores = list(map(int, input().split()))
print((sum(scores) / max(scores) * 100) / n)
'''

File: Baekjoon/Class2/09.1546.java
import java.io.*;
import java.util.StringTokenizer;

class Main {
    public static void main(String[] args) throws IOException {
        int n, m = 0, total = 0;
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;

        n = Integer.parseInt(br.readLine());
        st = new StringTokenizer(br.readLine());
        for(int i = 0; i < n; i++) {
            int score = Integer.parseInt(st.nextToken());
            total += score;
            if(score > m)
                m = score;
        }
        System.out.print(((float)total / m * 100) / n);
        br.close();
    }
}